database.go

```diff
@@ -1089,7 +1089,11 @@ func (db *datastore) GetPosts(c *Collection, page int, includeFuture, forceRecen
 	if !includeFuture {
 		timeCondition = "AND created <= " + db.now()
 	}
-	rows, err := db.Query("SELECT "+postCols+" FROM posts WHERE collection_id = ? AND pinned_position IS NULL "+timeCondition+" ORDER BY created "+order+limitStr, collID)
+	pinnedCondition := ""
+	if !includePinned {
+		pinnedCondition = "AND pinned_position IS NULL"
```

Comment:
I think you will need a space on the end, `AND pinned_position IS NULL `. Otherwise it will run into `timeCondition`.

Reply:
The space is included in the query string on line 1096. :)

Reply:
oh I totally missed that. :blush: 

not that I look though, does white space matter here? I mean extra. for example, if you have only `timeCondition` would the query having two spaces cause an error?

Reply:
It doesn't. You can see in the deleted row that it's possible to have a double space if `timeCondition` is empty, and SQL doesn't seem to care at all. :)

Reply:
sweet, thanks for teaching me :)